src/viam/app/data/client.py

```diff
@@ -80,15 +80,15 @@ async def tabular_data_by_filter(
         filter: Optional[Filter] = None,
         dest: Optional[str] = None,
     ) -> List[Mapping[str, Any]]:
-        """Filter and download tabular data
+        """Filter and download tabular data.
 
         Args:
-            filter (viam.app.data.Filter): When supplied, the tabular data will be
-            filtered based on the provided constraints. Otherwise, all data is returned.
-            dest (str): When supplied, the tabular data will be saved to the provided file path.
+            filter (viam.proto.app.data.Filter): Optional `Filter` specifying tabular data to retrieve. No `Filter` implies all tabular
```

Comment:
(q) have you confirmed that this change fixes the formatting issues that were showing up before?

Reply:
Working on this now! I haven't been able to get the relevant html files generated yet.